README.md

```diff
@@ -60,7 +63,35 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+sudo yum -y install epel-release
+sudo yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
+    python2-six \
+    python-gobject-base \
+    python-jsonschema \
+    python-setuptools \
+    python-ipaddress
+sudo echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
```

Comment:
How about recommending to run the script that we use for docker as well?

Reply:
If we specify it here explicitly we do not need to elaborate on how to run that script and where it is. It's not that costly and simple enough, so I think it is good enough.

For the future, it will be nice if we can put all this requirements into the setup.py (or call a script from there), so all this will be done as part of the installation.